templates/committee_manager/list_members.html.twig

```diff
@@ -182,13 +182,11 @@
                             {% endif %}
                         </td>
                         <td class="member-age">{{ member.age|default('~') }}</td>
-                        {% if is_canary_enabled() and is_supervisor %}
+                        {% if is_supervisor %}
                             <td class="member-phone">
-                                {% if membership.isVotingCommittee %}
-                                    <a href="tel:{{ member.phone|phone_number_format }}">
-                                        {{ member.phone|phone_number_format }}
-                                    </a>
-                                {% endif %}
+                                <a href="tel:{{ member.phone|phone_number_format }}">
+                                    {{ member.phone|phone_number_format }}
+                                </a>
```

Comment:
could you add a check if member.phone is not empty, please? otherwise it will crash